### PR TITLE
add dummy csv files, running full files is slow

### PR DESCRIPTION
**Pull request by stephenfabian**
Created dummy csv files to run tests more quickly.  Collabed over Zoom

### CODE REVIEW

**Comment by knestler** [comment]
great thanks!

**Review by knestler — DISMISSED** [review]

File: runner.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 require './lib/stat_tracker'
 
 
-game_path = './data/games.csv'
+game_path = './data/games_dummy.csv'
```
Comment (aleish-m) [inline runner.rb]: Lets do this on the spec fil vs runner so that our tests run faster but our runner is still the full file for the project

**Review by KevinT001 — APPROVED** [review]
Great thinking on creating smaller files for testing purposes!

File: runner.rb (hunk 1 above)
Comment (aleish-m) [reply runner.rb]: Thank you!

**Review by knestler — APPROVED** [review]

**Review by aleish-m — APPROVED** [review]

**Pull request merged**